os/pm/pm_changestate.c

```diff
@@ -242,6 +242,11 @@ int pm_changestate(enum pm_state_e newstate)
 		pm_changeall(newstate);
 #ifdef CONFIG_PM_METRICS
 		pm_metrics_update_changestate();
+#endif
+#ifdef CONFIG_LCD
+	if ((g_pmglobals.state != PM_STANDBY || newstate != PM_SLEEP) && (g_pmglobals.state != PM_SLEEP || newstate != PM_STANDBY)) {
```

Comment:
The check for controlling the LCD is here, but it is strange to check the state one more time for the value to actually control the LCD.

how about remove pm_lcd_get_power() and control lcd backlinght using argument in here?

Reply:
We can remove the checking in `pm_lcd_get_power` itself by using `PM_STATE->LCD_POWER` mapping array